packages/@azure/eventhubs/client/lib/eventHubClient.ts

```diff
@@ -5,7 +5,7 @@ import * as log from "./log";
 import { Delivery } from "rhea-promise";
 import {
   ApplicationTokenCredentials, DeviceTokenCredentials, UserTokenCredentials, MSITokenCredentials
-} from "ms-rest-azure";
+} from "@azure/ms-rest-nodeauth";
```

Comment:
We should probably update our examples as well to use this library instead of ms-rest-azure?